python/tests/basics.py

```diff
@@ -8,11 +8,15 @@
 
 # Load credentials from .env file
 api_key, org_id = sk.openai_settings_from_dot_env()
+# deployment_name, api_key, endpoint = sk.azure_openai_settings_from_dot_env()
```

Comment:
Any reason we don't want to test the azure openai functionality? Since the azure changes are in this pr, would be great to have a test for it too?